yourgov/pages/app/licences-and-permits/apply/mobile-food-vendor-permit/form/index.tsx

```diff
@@ -31,15 +38,120 @@ const Page: NextPageWithLayout = () => {
 							>
 								Back
 							</DirectionLink>
-							<PageTitle
-								title={formTitle}
-								introduction="Seek approval for your food van or truck to operate in our area."
-								callToAction={
-									<Text as="p" color="muted">
-										Takes around 20 minutes.
-									</Text>
-								}
-							/>
+							<Stack gap={2}>
+								<PageTitle
+									title={formTitle}
+									introduction="Seek approval for your food van or truck to operate in our area."
+									callToAction={
+										<Text as="p" color="muted">
+											Takes around 20 minutes.
+										</Text>
+									}
+								/>
+								<Details label="What you need to apply for a permit" iconBefore>
+									<Stack gap={1} alignItems="start">
+										<Text as="p">
+											To complete this application you will need to provide:
+										</Text>
+										<UnorderedList>
+											<ListItem>
+												vehicle registration details and vehicle plans
+											</ListItem>
+											<ListItem>operational plan of management</ListItem>
+											<ListItem>food safety supervisor certificate</ListItem>
+											<ListItem>
+												names of any employees who will be handling food
+											</ListItem>
+											<ListItem>
+												once you have confirmed business and employee details we
```

Comment:
Does this line need to make sense?

```
To complete this application you will need to provide:
// ...
once you have confirmed business and employee details we will provide a checklist of required documents.
```

Reply:
Good question. I just copied it from Paul's design. I have a feeling that should be another paragraph after the list. Will change

Reply:
Done